contents.json

```diff
@@ -1139,6 +1139,12 @@
       "description": "Compose beautiful command line interfaces.",
       "homepage": "https://github.com/kylef/Commander",
       "tags": ["linux"]
+    }, {
+      "title": "Guaka",
+      "category": "command-line",
+      "description": "The smart and beautiful (POSIX compliant) command line framework",
```

Comment:
@thii I didn't noticed at last time, could you add a full stop/period for description?